rootfs_linux.go

```diff
@@ -24,9 +25,15 @@ func setupRootfs(config *configs.Config, console *linuxConsole) (err error) {
 		return newSystemError(err)
 	}
 	for _, m := range config.Mounts {
+		if err := mountCmd(m.PremountCmd); err != nil {
```

Comment:
Do you think it's better to setup some special env variables like rootfs path ( similar to how lxc does it ) so commands can be self-aware of the container rootfs, or is it enough for the caller ( i.e., docker ) to pass them in as arguments ?